code/HubPage.php

```diff
@@ -20,6 +21,8 @@ public function getCMSFields()
     {
         $fields = parent::getCMSFields();
 
+        $fields->insertAfter('URLSegment', new TextareaField('Description', 'Description'));
```

Comment:
Suggest adding a setDescription on this to provide the content editor some context of how they use this.